gi_core/CMakeLists.txt

```diff
@@ -19,22 +19,60 @@ project(gi_core)
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(common)
+include(CPM)
 
-find_c_and_cpp_files("${CMAKE_CURRENT_SOURCE_DIR}/include" gi_core_includes)
+# GI_CORE
+
+find_package(Boost COMPONENTS system filesystem unit_test_framework REQUIRED)
+find_package(OpenMP)
+
+CPMFindPackage(
+  NAME libsimdpp
+  GITHUB_REPOSITORY TheTryton/libsimdpp
+  VERSION 2.1.1
+)
+
+CPMFindPackage(
+  NAME vexcl
```

Comment:
What is that package?
It seems that current changes inserted some dependencies on the project. This should be clearly stated in main README file.

Reply:
This library will be used for running gamma index on gpu - a lot better that calling OpenCL/CUDA directly.